public/data/varghesejose2020.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Varghese Jose",
+  "type": "personal",
+  "bio": "Open Source Contributor||Full Stack Developer||FOSS Enthusiast",
```

This is a small suggestion, because I think it'll look weird on the site
```suggestion
  "bio": "Open Source Contributor | Full Stack Developer | FOSS Enthusiast",
```

---

:+1: